src/pages/workspace/companyCards/assignCard/TransactionStartDateStep.tsx

```diff
@@ -37,7 +41,13 @@ function TransactionStartDateStep() {
             });
             return;
         }
-        setAssignCardStepAndData({currentStep: CONST.COMPANY_CARD.STEP.CARD});
+        const backTo = route?.params?.backTo;
```

Comment:
AFAIR, we deprecated `backTo`, is there another way to handle this without `backTo`?